src/app/components/Image/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const getText = block => block.model.blocks[0].model.blocks[0].model.text;
+
+const Image = ({ blocks }) => {
+  const subBlocks = blocks[0].model.blocks;
```

Comment:
This seems to be passing the root blocks object, but we should only pass the `image` block: https://github.com/bbc/simorgh/blob/latest/data/prod/scenario-27.json#L46-L110

Reply:
OK. I was using the example in the Headline component as reference: https://github.com/bbc/simorgh/blob/latest/src/app/components/Headline/index.jsx#L4  If we want to update this here, we should likely update the Headline component as well.